dist/flags.py

```diff
@@ -142,6 +142,7 @@
     'stat' : [
         'STAT_CLEAR',
         'STAT_JSON',
+        'STAT_JSON_PRINT_TABLES',
```

Comment:
This new flag feels a bit out of place. The others correspond 1-1 with options available via the public statistics configuration options, wheras this tracks internal state during a single stats pass.

I'm kind of concerned that the code is setting and clearing the flag that's in the global `WT_CONNECTION` handle, when it's relevant for a single pass of statistics gathering which is at the per-cursor granularity. i.e: I think it's possible to have two cursors gathering stats at the same time, and I think their setting/clearing of the new flag in this set of flags would interfere with each other.